balancer-js/src/modules/data/pool-joinExit/repository.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any */
+import { PoolJoinExit, PoolJoinExitAttributes } from './types';
+import { BalancerSubgraphRepository } from '@/modules/subgraph/repository';
+import {
+  JoinExit_OrderBy,
+  OrderDirection,
+  SubgraphJoinExitFragment,
+} from '@/modules/subgraph/generated/balancer-subgraph-types';
+
+export class PoolJoinExitRepository extends BalancerSubgraphRepository<
+  PoolJoinExit,
+  PoolJoinExitAttributes
+> {
```

Comment:
We have Findable<T> / Searchable<T> interfaces for repository classes. do you mind trying to use it here? it makes testing easier and is more predictable to use.

Reply:
True, you are right, didn't notice we had an interface like that. Sorry for that.